User-Help/Scoring

```diff
@@ -0,0 +1,22 @@
+
+# Question scoring and ordering
+
+## Overview 
+
+Communities need a way to identify and reward quality contributions to their sites. They also need a way to sort post threads so higher quality content is shown first. Codidact solves this problem by assigning each post a score based on a ratio of upvotes to total votes, weighted to favor votes with fewer downvotes. For example, a post with seven upvotes will be shown higher than a post with nine upvotes and two downvotes. This method gives a slight boost to contributions that are consistently viewed positively over more controversial answers that gather both upvotes and downvotes. 
```

Comment:
Since this is for the help, it feels a little odd to be talking about communities and what "they" need here.  Maybe something like: "On Codidact, answers to questions are sorted by their scores, so that you'll see the best-received answers first.  Each post is assigned a score based on a ratio... (etc)".

Reply:
I've been trying to include some of the context of _why_ things are done a certain way in the user help, rather than just give a "here's how it is." Maybe that is better kept elsewhere?

Reply:
I think "why" is good, but maybe lead with "what" and then do "why"?  For a blog post or similar leading with the problem and leading the reader through the reasoning is fine; for help I think we should lead with the info that somebody clicking that topic in the help was probably looking for.  Does that make sense?